tests/mocks/MockClock.hh

```diff
@@ -15,7 +15,7 @@ public:
     MockClock() noexcept : m_Override(this)
     {
         // force GMT timezone for unit tests
-        _putenv("TZ=GMT");
+        Expects(_putenv("TZ=GMT") == 0);
```

Comment:
`_putenv `has a `_Check_return_` annotation which is the SAL equivalent to the C++ `[[nodiscard]]` attribute. Except `_Check_return_` is only active during code analysis and `[[nodiscard]]` is always active.

Reply:
Done